modules/wireguard.py

```diff
@@ -1,16 +1,49 @@
 # Copyright 2024 dhtech
-#
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file
+
 import lib
+import sqlite3
+import os
+import ipcalc
+import sys
```

Comment:
is sys used?

Reply:
Sys is indeed not used anymore, I've removed it.